flake.nix

```diff
@@ -141,6 +143,17 @@
                   --replace /bin/bash "${bash}/bin/bash"
               done
           '';
+          configurePhase =
+            ''
+            export LIBCLANG_PATH="${pkgs.llvmPackages.libclang.lib}/lib"
+            export BINDGEN_EXTRA_CLANG_ARGS="$(< ${pkgs.stdenv.cc}/nix-support/libc-crt1-cflags) \
+              $(< ${pkgs.stdenv.cc}/nix-support/libc-cflags) \
+              $(< ${pkgs.stdenv.cc}/nix-support/cc-cflags) \
+              $(< ${pkgs.stdenv.cc}/nix-support/libcxx-cxxflags) \
+              $NIX_CFLAGS_COMPILE"
+            export OUT=${placeholder "out"}
+            echo $OUT
```

Comment:
What does this do?

Reply:
Ah, that was me trying to understand some things. I'll circle around and clean it up.